packages/polaris-viz/src/components/SparkLineChart/tests/Chart.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('<Chart />', () => {
     const offsetLeft = 100;
     const offsetRight = 50;
     const margin = 2;
-    const mockWidth = 0;
+    const mockWidth = 600;
```

Comment:
Confused why we have Sparkline test changes on this PR. Would be good to add a comment here to explain or in the PR description in the future.

Reply:
Adding a little explanation here: https://github.com/Shopify/polaris-viz/pull/1771#issuecomment-2514953274